mlos_bench/mlos_bench/services/config_persistence.py

```diff
@@ -661,42 +674,44 @@ def load_services(
         service : Service
             A collection of service methods.
         """
-        _LOG.info("Load services: %s parent: %s", json_file_names, parent.__class__.__name__)
+        _LOG.info("Load services: %s parent: %s", jsons, parent.__class__.__name__)
         service = Service({}, global_config, parent)
-        for fname in json_file_names:
-            config = self.load_config(fname, ConfigSchema.SERVICE)
+        for json in jsons:
+            config = self.load_config(json, ConfigSchema.SERVICE)
             service.register(self.build_service(config, global_config, service).export())
         return service
 
-    def _load_tunables(
+    def load_tunables(
```

Comment:
Make public for testing.
Didn't add it to the SupportsConfigLoadingType though, since it's not used anywhere else atm.